src/main/java/io/cryostat/net/OpenShiftAuthManager.java

```diff
@@ -296,6 +311,23 @@ public AuthenticationScheme getScheme() {
         }
     }
 
+    private Boolean deleteToken(String token) throws IOException, TokenNotFoundException {
```

Comment:
I think the return type here can be an unboxed `boolean`, since the `Boolean.FALSE` check lower down takes care of the nullability case - no need to potentially propagate a `null` any further.